src/main/java/Group15/Workout.java

```diff
@@ -0,0 +1,34 @@
+package Group15;
+
+import Group15.Api.Exercise;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Workout {
+    private List<Exercise> exercises = new ArrayList<>();
+
+    public Workout() {}
+
+    public void addExercise(Exercise exercise) {
+        exercises.add(exercise);
+    }
+
+    public void swapExercise(Exercise toRemove, Exercise toAdd) {
```

Comment:
You forgot to return inside the if you stupid idiot

Reply:
:)

Reply:
How did that go through review, wtf :)